<event logger="KPIWeb.LogWriter" timestamp="2015-02-23T17:14:27.0010245+03:00" level="ERROR" thread="18" domain="/LM/W3SVC/8/ROOT-1-130691743583838119" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T08:40:20.7550897+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130692300025186577" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T08:47:53.5016995+03:00" level="ERROR" thread="33" domain="/LM/W3SVC/8/ROOT-1-130692300025186577" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T08:58:54.3166748+03:00" level="ERROR" thread="50" domain="/LM/W3SVC/8/ROOT-1-130692300025186577" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T09:09:48.4036622+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692317715396976" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T09:11:49.3055774+03:00" level="ERROR" thread="20" domain="/LM/W3SVC/8/ROOT-1-130692317715396976" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T09:38:42.3965837+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692335003103205" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T09:55:29.6256821+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130692345097895475" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T10:00:17.7049045+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692348001216735" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T10:03:37.7293051+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130692349930158616" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T10:05:14.6249205+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130692350994606938" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T10:06:43.6424483+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692351810380086" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T10:08:45.5927413+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130692353103983146" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T11:00:10.5790161+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692383924209775" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T11:02:54.9113526+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692385636481328" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T11:17:43.241399+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130692394383479752" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T12:09:56.1798044+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130692425403706123" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T12:33:32.4881622+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130692440019181474" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T12:35:21.4019429+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130692441099219268" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T12:40:26.194651+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130692443975930151" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T12:57:31.1980513+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692453924266897" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T13:00:16.6625153+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130692456026017111" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T13:46:22.8767337+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692483687859277" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T14:10:50.6126834+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692498299685026" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T14:39:53.799388+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-2-130692515833087879" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T14:39:59.0036856+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-2-130692515833087879" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T14:40:21.0449463+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-2-130692515833087879" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T14:40:25.9842288+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-2-130692515833087879" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T14:40:54.9158836+03:00" level="ERROR" thread="5" domain="/LM/W3SVC/8/ROOT-2-130692515833087879" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T16:16:16.5661433+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T16:16:30.3789334+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T16:16:44.604747+03:00" level="ERROR" thread="17" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T16:26:40.4538276+03:00" level="ERROR" thread="36" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T16:27:18.696015+03:00" level="ERROR" thread="24" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T16:53:11.5248317+03:00" level="ERROR" thread="105" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:00:03.4343916+03:00" level="ERROR" thread="14" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:14:22.9745545+03:00" level="ERROR" thread="52" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:15:30.3684092+03:00" level="ERROR" thread="71" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:16:49.7109473+03:00" level="ERROR" thread="59" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:17:57.856845+03:00" level="ERROR" thread="18" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:35:52.9123347+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:36:51.8887079+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:37:09.0746909+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:42:37.9305004+03:00" level="ERROR" thread="93" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:43:38.1519449+03:00" level="ERROR" thread="38" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:44:03.3793878+03:00" level="ERROR" thread="82" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:44:31.9050194+03:00" level="ERROR" thread="31" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T17:47:51.1464153+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/8/ROOT-1-130692573581060875" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:05:08.7695673+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130692782807799663" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:48:15.8225382+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130692808811316980" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:48:31.9804624+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130692808811316980" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:52:37.6375132+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692811404995329" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:53:08.0632534+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692811404995329" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:53:26.379301+03:00" level="ERROR" thread="14" domain="/LM/W3SVC/8/ROOT-1-130692811404995329" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:54:43.1846941+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130692811404995329" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T22:56:20.8772818+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692813691626117" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T23:44:32.9736066+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692842596968472" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T23:51:06.8591356+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130692846540134008" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-24T23:51:44.4882878+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692846941796982" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:05:43.6762867+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130692855289294432" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:06:46.814898+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692855928751007" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:07:36.9147635+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130692856430629712" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:08:46.9887715+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692857077726724" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:09:33.2154155+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692857598046485" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:12:09.7383681+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130692859122683689" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:13:55.423413+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130692860216926276" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:18:40.1186966+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130692863018746531" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T00:33:28.1498815+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130692871922838545" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T01:02:02.4340847+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130692888975036587" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T01:16:05.6362514+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130692897532972186" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T08:50:03.4502462+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130693169837691206" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T09:14:25.4888701+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130693184492549415" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T09:19:31.3173625+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130693187520202587" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T10:16:53.6897908+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/6/ROOT-1-130693221868352548" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T10:34:40.1087865+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/6/ROOT-1-130693227927779128" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T10:35:18.1199606+03:00" level="ERROR" thread="28" domain="/LM/W3SVC/6/ROOT-1-130693227927779128" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T10:38:00.3372389+03:00" level="ERROR" thread="15" domain="/LM/W3SVC/6/ROOT-1-130693227927779128" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T10:39:22.6499469+03:00" level="ERROR" thread="33" domain="/LM/W3SVC/6/ROOT-1-130693227927779128" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T10:40:52.5190872+03:00" level="ERROR" thread="36" domain="/LM/W3SVC/6/ROOT-1-130693227927779128" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T11:24:40.1853811+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/6/ROOT-1-130693261793306125" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T12:37:45.1151847+03:00" level="ERROR" thread="5" domain="/LM/W3SVC/6/ROOT-1-130693306387056742" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T13:43:47.7507971+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/6/ROOT-1-130693345891145873" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T14:31:35.2438084+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/6/ROOT-1-130693374753026678" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T16:19:24.1598092+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/6/ROOT-1-130693439436046335" username="HP_ProBook_4340\SiV@S"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\SiV@S\Desktop\Repo\Projects.git\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="HP_ProBook_4340" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-25T16:42:16.1557495+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130693453057220088" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T10:14:32.545728+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130694084445691279" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T10:31:47.8779457+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130694094874087749" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T10:32:01.2667114+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130694094874087749" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T10:47:42.5075473+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130694104301976993" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T10:50:05.3807192+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130694105833944617" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T14:58:27.8154146+03:00" level="ERROR" thread="14" domain="/LM/W3SVC/8/ROOT-1-130694254814179048" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T14:59:53.4953152+03:00" level="ERROR" thread="19" domain="/LM/W3SVC/8/ROOT-1-130694254814179048" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-26T15:04:54.6895425+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130694258800077028" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T10:02:27.5467494+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130694941200681777" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T10:52:38.603972+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130694971380097941" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T10:54:37.5647762+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130694971380097941" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T10:59:03.8460066+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130694975284801277" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:01:46.2122934+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130694976904073894" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:02:52.555088+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130694976904073894" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:13:37.4409733+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130694983863251936" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:23:58.6465043+03:00" level="ERROR" thread="15" domain="/LM/W3SVC/8/ROOT-1-130694989125372912" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:32:59.5884444+03:00" level="ERROR" thread="21" domain="/LM/W3SVC/8/ROOT-1-130694992101283125" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:38:51.6975839+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130694999078442195" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:39:45.4696595+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130694999692197300" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:47:05.644836+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130695004055436863" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:47:16.2084402+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130695004055436863" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T11:54:20.9557344+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130695008376914038" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T12:11:09.6154264+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130695018513103794" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T16:48:59.0539173+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130695184875829733" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T16:49:17.702984+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/8/ROOT-1-130695184875829733" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T16:49:54.6090949+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130695184875829733" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T16:52:59.4776687+03:00" level="ERROR" thread="21" domain="/LM/W3SVC/8/ROOT-1-130695184875829733" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T16:53:09.1952246+03:00" level="ERROR" thread="18" domain="/LM/W3SVC/8/ROOT-1-130695184875829733" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T17:00:14.6765607+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130695191995736969" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-27T17:03:12.2917197+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130695193718835524" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T14:06:47.4861466+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130695951888260793" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T14:23:02.6812586+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130695961685788338" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T14:39:40.2537288+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130695971687877086" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T15:30:12.2469793+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130696001985621966" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T15:32:22.9064526+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130696003268555346" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T15:42:27.2180173+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130696009335022328" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T15:49:41.8958794+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130696013675540591" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T16:59:18.7912822+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130696055254543754" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 49KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T16:59:25.5166669+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130696055254543754" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T17:00:26.893892+03:00" level="ERROR" thread="18" domain="/LM/W3SVC/8/ROOT-1-130696055254543754" identity="admin" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Register.CreateUser_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Register.aspx.cs:строка 151KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T17:01:15.0511766+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130696055254543754" identity="admin" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Register.CreateUser_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Register.aspx.cs:строка 151KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T17:06:19.9458636+03:00" level="ERROR" thread="15" domain="/LM/W3SVC/8/ROOT-1-130696059149801478" identity="admin" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.Register.CreateUser_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Register.aspx.cs:строка 149KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-28T17:09:14.8795911+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130696060955641985" identity="admin" username="V-PC\V"><message>System.InvalidOperationException: Не удается выполнить операции Create, Update и Delete для "Table(IndicatorsAndUsersMapping)", так как он не содержит первичного ключа.    в System.Data.Linq.Table`1.CheckReadOnly()    в System.Data.Linq.Table`1.InsertOnSubmit(TEntity entity)    в KPIWeb.Account.Register.CreateUser_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Register.aspx.cs:строка 186KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-01T10:17:51.6093532+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130696678561809261" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-01T10:22:07.5578718+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130696680769825829" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
